src/theme/MDXComponents/Pre.tsx

```diff
@@ -41,6 +43,15 @@ const Pre = ({ children, className }: CodeProps) => {
         ".hljs",
       ];
 
+      // If copyText includes at least one CommandLine, the intention is for
+      // users to copy commands and not example outputs (CodeLines). If there
+      // are no CommandLines, it is fine to copy the CodeLines.
+      if (copyText.getElementsByClassName(commandStyles.line).length > 0) {
```

Comment:
This may not work in production when Docusaurus adds hashes to the ends of class names.